src/models/slices/vertical.ts

```diff
@@ -1,11 +1,34 @@
-import { VerticalSearchResponse, AutocompleteResponse, VerticalResults } from '@yext/answers-core';
+import {
+  AppliedQueryFilter,
+  AutocompleteResponse,
+  DisplayableFacet,
+  Result,
+  SearchIntent,
+  SortBy,
+  Source,
+  VerticalResults
+} from '@yext/answers-core';
+
+export interface AllResultsForVertical {
```

Comment:
Just refreshing my memory, this is what we decided would be best with Rose, correct? Later on, we may make a setting or something that automatically hydrates the top-level facets and search intents from this when there are no results.

Reply:
Ya, this what we discussed we'd do for now. The only difference is that I changed the model name from `AllResults` to `AllResultsForVertical` since that's what the API uses